web/themes/custom/vote_gov/templates/node/node--page--full.html.twig

```diff
@@ -0,0 +1,105 @@
+ *     name will often be in a short form of the human readable label.
+ *   - node--view-mode-[view_mode]: The View Mode of the node; for example, a
+ *     teaser would result in: "node--view-mode-teaser", and
+ *     full: "node--view-mode-full".
+ *   The following are controlled through the node publishing options.
+ *   - node--promoted: Appears on nodes promoted to the front page.
+ *   - node--sticky: Appears on nodes ordered above other non-sticky nodes in
+ *     teaser listings.
+ *   - node--unpublished: Appears on unpublished nodes visible only to site
+ *     admins.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - content_attributes: Same as attributes, except applied to the main
+ *   content tag that appears in the template.
+ * - author_attributes: Same as attributes, except applied to the author of
+ *   the node tag that appears in the template.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - view_mode: View mode; for example, "teaser" or "full".
+ * - teaser: Flag for the teaser state. Will be true if view_mode is 'teaser'.
+ * - page: Flag for the full page state. Will be true if view_mode is 'full'.
+ * - readmore: Flag for more state. Will be true if the teaser content of the
+ *   node cannot hold the main body content.
+ * - logged_in: Flag for authenticated user status. Will be true when the
+ *   current user is a logged-in member.
+ * - is_admin: Flag for admin user status. Will be true when the current user
+ *   is an administrator.
+ *
+ * @see template_preprocess_node()
+ *
+ */
+#}
+{% set classes = [
+  'node',
+  'node--type-' ~ node.bundle|clean_class,
+  node.isPromoted() ? 'node--promoted',
+  node.isSticky() ? 'node--sticky',
+  not node.isPublished() ? 'node--unpublished',
+  view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+] %}
+
+<div class="bg-lightblue">
+  <div class="grid-container">
+    <div class="grid-row grid-gap">
+      <div class="tablet:grid-col-4">
+      </div>
+      <div class="tablet:grid-col-8 right-shift">
+        <section class="voter-info page-type--basic">
+          <div class="about-page">
+            {# Title banner and main content #}
+            {{ content }}
+            {# Homepage button #}
+            <a href="/">
+              <button type="button" class="usa-button">
+              Get Registered
+              </button>
+            </a>
+          </div>
+        </section>
+        {# Go back link - where should this go? #}
+        <p class="arrow-text"><a href="/" title="Vote.gov" data-test="back-button"><span class="arrow direction"></span>Go back</a></p>
```

Comment:
This needs to move into a partial template that we can include as needed on other pages
```
{# Go back link - link to homepage #}
        <p class="arrow-text"><a href="/" title="{{ 'Vote.gov' | t }}" data-test="back-button"><span class="arrow direction"></span>{{ 'Go back' | t }}</a></p>
```